source/user/index.php

```diff
@@ -1,4 +1,7 @@
 <?php
+/*
+ * Payment Service
```

Comment:
Hi @Lagrida , is this comment about payment supposed to be here in user source?

Reply:
sorry, it's not intentional error

Reply:
No worries, please remove and recommit. Also give more meaningful commit messages.
something like "removed unintentional comment from user source"

Reply:
okey thank you, i will pull again

Reply:
Not pull, I think just make your changes and commit then push. There is a conflict here we would fix tomorrow.

Reply:
okey then